azure-pipelines.yml

```diff
@@ -178,182 +232,12 @@ jobs:
-            exit $status
-
-  - job: build_macos_library
-    displayName: Build macOS Library
-    pool:
-      vmImage: macos-12
-    steps:
-      # if this is a tagged build, then update the version number
-      - powershell: |
-          $buildSourceBranch = "$(Build.SourceBranch)"
-          $tagVersion = $buildSourceBranch.Substring($buildSourceBranch.LastIndexOf("/") + 1)
-          Write-Host("Branch = $buildSourceBranch, Version = $tagVersion");
-          Write-Host ("##vso[task.setvariable variable=NugetPackageVersion;]$tagVersion")
-        displayName: Set NuGet Version to Tag Number
-        condition: startsWith(variables['Build.SourceBranch'], 'refs/tags/')
-
-      - task: CmdLine@2
-        displayName: 'Set Xcode Version'
-        inputs:
-          script: echo '##vso[task.setvariable variable=MD_APPLE_SDK_ROOT;]'/Applications/Xcode_$(Xcode13Version).app;sudo xcode-select --switch /Applications/Xcode_$(Xcode13Version).app/Contents/Developer
-     
-      - task: UseDotNet@2
-        displayName: Install .NET $(TOOLKIT_NET6_VERSION)
-        inputs:
-          packageType: 'sdk'
-          version: '$(TOOLKIT_NET6_VERSION)'
-
-      - task: CmdLine@2
-        displayName: 'Install .NET MAUI Workload $(TOOLKIT_NET6_VERSION)'
-        inputs:
-          script: dotnet workload install maui --from-rollback-file $(DotNetMauiRollbackFile) --source https://api.nuget.org/v3/index.json
-
-      - task: CmdLine@2
-        displayName: 'Build CommunityToolkit.Maui.Analyzers'
-        inputs:
-          script: 'dotnet build -c Release $(PathToCommunityToolkitAnalyzersCsproj)'
-
-      - task: CmdLine@2
-        displayName: 'Build CommunityToolkit.Maui.Analyzers.CodeFixes'
-        inputs:
-          script: 'dotnet build -c Release $(PathToCommunityToolkitAnalyzersCodeFixCsproj)'
-
-      - task: CmdLine@2
-        displayName: 'Build CommunityToolkit.Maui.SourceGenerators'
-        inputs:
-          script: 'dotnet build -c Release $(PathToCommunityToolkitSourceGeneratorsCsproj)'
-
-      - task: CmdLine@2
-        displayName: 'Run CommunityToolkit.Maui.Analyzers.UnitTests'
-        inputs:
-          script: 'dotnet test $(PathToCommunityToolkitAnalyzersUnitTestCsproj) -c Release'
-
-      - task: CmdLine@2
-        displayName: 'Run CommunityToolkit.Maui.UnitTests'
-        inputs:
-          script: 'dotnet test $(PathToCommunityToolkitUnitTestCsproj) -c Release'
-
-      - task: CmdLine@2
-        displayName: 'Pack CommunityToolkit.Maui.Core NuGet'
-        inputs:
-          script: 'dotnet pack $(PathToCommunityToolkitCoreCsproj) -c Release -p:PackageVersion=$(NugetPackageVersion)'
-
-      - task: CmdLine@2
-        displayName: 'Pack CommunityToolkit.Maui NuGet'
-        inputs:
-          script: 'dotnet pack $(PathToCommunityToolkitCsproj) -c Release -p:PackageVersion=$(NugetPackageVersion)'
-
-  - job: build_macos_sample
```

Comment:
I think it's important to continue building the sample app using the latest .NET Version. 

I think it's also important for us to pin our releases to .NET 7.0 v1, like we currently do for .NET 6.0 v1. 

Let's not delete the `build macOS sample using latest .NET version` build step.

Reply:
I didn't delete it. it is still there ( you can see it in the checks).

Reply:
I removed duplicated code for windows and macos build to make sure we use the same pipeline steps for both platforms. they run in matrix